zaza/openstack/charm_tests/manila/tests.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Encapsulate Manila testing."""
+
+
+from manilaclient import client as manilaclient
+
+import zaza.openstack.charm_tests.test_utils as test_utils
+import zaza.openstack.utilities.openstack as openstack_utils
+
+
+class ManilaTests(test_utils.OpenStackBaseTest):
+    """Encapsulate Manila  tests."""
+
+    RESOURCE_PREFIX = 'zaza-manilatests'
+    INSTANCE_USERDATA = """#cloud-config
+packages:
+- nfs-common
+"""
```

Comment:
Formatting looks a bit strange?

Reply:
it's a multiline cloud-config string - I think that the #cloud-config line has to be on the first line of the user-data, and then, indentation matters, so `packages:` has to be on the left-most column